wekesa931/Questioner
# [finishes #163096709] creates the fetch-meetup endpoint

#### What does this PR do?
It creates the fetch-meetup api endpoint
#### Description of Task to be completed?
To create the fetch-meetup api endpoint path:
```
GET    /api/v1/meetups/<int:meetup_id>
```
This endpoint returns the required details of a meetup which are:
```
{
    “id” : Integer ,
    “createdOn” : Date ,
    “location” : String ,
    “images” : [String, String] , // OPTIONAL: URL to the image location
    “topic” : String ,
    “happeningOn” : Date , // when the meetup is holding
    “Tags” : [String, String, ....] ,
}
```
#### How should this be manually tested?
clone the repository to your local machine
```
$ git clone https://github.com/wekesa931/Questioner.git
```
Set up the virtual environment
```
$ pip install virtualenv
$ virtualenv venv
$ venv source bin/activate
```
run the application
```
$ python run.py
```
Test the endpoint in postman as follows:
In postman, go to path
```
GET    /api/v1/meetups/<int:meetup_id>
```
Note that this will show when you have already created some meetups
#### What are the relevant pivotal tracker stories?
    #163096709
#### Screenshots (if appropriate)
![meetupid](https://user-images.githubusercontent.com/31995387/50922114-ba7ee000-145a-11e9-9af1-4b7eb37d9e77.png)
